Documentation/doc/scripts/generate_how_to_cite.py

```diff
@@ -85,7 +85,7 @@
 <link rel="icon" type="image/png" href="../Manual/g-196x196-doc.png"/>
 <meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
 <meta http-equiv="X-UA-Compatible" content="IE=9"/>
-<link href="stylesheet.css" rel="stylesheet" type="text/css" />
+<link href="cgal_stylesheet.css" rel="stylesheet" type="text/css" />
```

Comment:
That change is wrong. The file `stylesheet.css` is used and should be deduplicated as well. Here is the right patch:

```suggestion
<link href="stylesheet.css" rel="stylesheet" type="text/css" />
<link href="cgal_stylesheet.css" rel="stylesheet" type="text/css" />
```

Reply:
Oops! I am wrong.

Reply:
That change is okay. I tested it.